app/bundles/CacheBundle/README.md

```diff
@@ -0,0 +1,163 @@
+# mautic-cache-plugin
+
+Enables PSR-6 and PSR-16 caching. Check: [Symfony Cache Component](https://symfony.com/doc/3.4/components/cache.html)
+
+## Installation
+
+Install this plugin as submodule into plugins directory.
```

Comment:
@hluchas please remove this readme. I've moved the docs to https://github.com/mautic/developer-documentation/pull/179 and updated them. The installation steps are not up to date at the moment. It'll be easier to keep the docs up to date in one place only.

Reply:
removed ...